spring.application.name=changebook


spring.cloud.gcp.sql.database-name=change_book
spring.cloud.gcp.sql.instance-connection-name=tonal-history-267816:southamerica-east1:change-book-databases

# So app starts despite "table already exists" errors.
spring.datasource.continue-on-error=true

# Enforces database initialization
spring.datasource.initialization-mode=always

spring.datasource.username=root
spring.datasource.password=pucxyx-9pUgty-pycgoz
spring.cloud.gcp.sql.credentials.location=file:src/main/resources/cloud-credentials/ChangeBook-efcb83b3291c.json

spring.datasource.hikari.connectionTimeout=20000
spring.datasource.hikari.maximumPoolSize=5

spring.jpa.properties.hibernate.default_schema=PUBLIC
spring.jpa.show-sql=true
server.port=8080

#Application
message-locator.service-name=${message_locator_service_name:${spring.application.name}}
message-locator.service-id=${message_locator_service_id:1}
jwt.secret=ChangeBookAuth

#spring.jpa.hibernate.ddl-auto=update
